content/community/changelog/app-frontend/v3/whats-new/_index.en.md

```diff
@@ -3,6 +3,40 @@ title: What's new
 description: Overview of changes introduced in v3 of app frontend.
 toc: true
 ---
+
+## 3.27.0 (2022-02-01) - Show app owner in header
+App owner will now be displayed in the app header to improve visibility to who owns the app.
+
+Related to issue [#7227](https://github.com/Altinn/altinn-studio/issues/7227).
```

Comment:
```suggestion
Issue [#7227](https://github.com/Altinn/altinn-studio/issues/7227).
```

For å være konsekvent med navngivning tidligere for saker som løser hele issues.